wildcard-pair.gemspec

```diff
@@ -5,7 +5,7 @@
 
 Gem::Specification.new do |spec|
   spec.name          = "wildcard-pair"
-  spec.version       = "0.3.1"
+  spec.version       = "0.3.2"
```

Comment:
can we make this 0.4.1 just to line up to what we're using for Java (i should have made my initial keywords java sdk impl a patch update, not up to 0.4 - so my bad)